org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/debug/node/AttachTab.java

```diff
@@ -51,20 +65,47 @@ public void createControl(Composite parent) {
 			setDirty(true);
 			updateLaunchConfigurationDialog();
 		});
+		
+		new Label(resComposite, SWT.NONE).setText(Messages.RunProgramTab_workingDirectory);
+		Composite workingDirectoryComposite = new Composite(resComposite, SWT.NONE);
```

Comment:
In which case do we need it to be set? So far, it's been working well without it.

Reply:
Only for web app debuging in Firefox and Chrome, not Node.js. I'll move it into the child classes.